sdks/python/apache_beam/runners/worker/sdk_worker_main.py

```diff
@@ -239,24 +241,27 @@ def _parse_pipeline_options(options_json):
   return PipelineOptions.from_dictionary(_load_pipeline_options(options_json))
 
 
-def _get_state_cache_size(experiments):
-  """Defines the upper number of state items to cache.
-
-  Note: state_cache_size is an experimental flag and might not be available in
-  future releases.
+def _get_state_cache_size(options, experiments):
+  """Defines the maximum size of the cache in megabytes.
```

Comment:
there is an inaccuracy between the docstring and returned value. let's add units to the function and variable names, such as 

`_get_state_cache_size_bytes`

`state_cache_size_mb`